packages/components/link/src/link.visualroute.js

```diff
@@ -35,5 +35,8 @@ export const component = () => (
     <Spec label="intlMessage">
       <Link to="/" intlMessage={intlMessage} />
     </Spec>
+    <Spec label="white">
+      <Link to="/" tone={'inverted'} />
```

Comment:
does this have to be `{''}`? why not use double quotes directly?

Reply:
this is converted to single quotes by prettier